lib/plexy/config.ex

```diff
@@ -15,22 +15,24 @@ defmodule Plexy.Config do
   ran at runtime.
   """
 
+  alias __MODULE__
```

Comment:
what does this do / why is it needed?

Reply:
I _think_ thats what allows you to use `MyOtpApp.Config` instead of `Plexy.Config`

Reply:
and allows him to call `Config` throughtout this file without calling `Plexy.Config`

Reply:
This was required by Credo, not my choice.